Adds a new Ice Planet ruin, the Hydroponics Lab

iwishforducks opened the pull request:
## About The Pull Request

Adds a new Ice Planet ruin, the Hydroponics Lab. Located deep within a mountain, this old and rusted laboratory focuses on growing experimental and deadly plants.

![2022 07 12-16 45 36](https://user-images.githubusercontent.com/65363339/178592061-cb3ed0eb-fa1c-47a3-9711-cd2388a50ac2.png)

The main loot is as follows:

- Two M1911 handguns + one magazine in total
- One flamethrower
- Security winter coat
- Security jumpsuit
- Jackboots
- A variety of botany tools including a botany locker
- Tools + multitool
- Biogenerator, seed extractor
- Hydroponic trays
- Bottles of ammonia, diethylamine, ez-nutrient, unstable mutagen, left-4-zed, robust harvest, sodium and ethanol
- Replica pod seed
- Killer tomato seed
- Kudzu seed
- A variety of other plants and seeds
- Biosuit

The katana located in the walls is a replica. I hope some poor sap looks at this PR, sees the katana, and goes "OMG REAL KATANA!!!" and wastes their time trying to tear down the walls.

## Why It's Good For The Game

Ice ruins are lacking and we don't have any hydroponics themed ruins on the ice just yet.

More ruins = good

## Changelog
:cl:
add: Added a new ruin on Ice Planets, the Hydroponics Lab.
/:cl:

iwishforducks committed:
Initial commit

- Adds the map file
- Indexes the ruin
- Adds an area
- Adds paper fluff items

iwishforducks committed:
Loot + enemy adjustments

- Removes the insulated gloves located in the bathroom
+ Adds another venus human trap

iwishforducks committed:
Reduces magazine count

Removes one magazine from the security office and removes the one on the ground. Totals to only one single handgun magazine.

iwishforducks committed:
Made things look nicer

Tech floor decals, colored rims, rusted floors, etc. Just generally nicer things.

iwishforducks committed:
Fixes quotations

Varedited something wrong. Whoops.

iwishforducks committed:
pushes the dme file

New files require it to be marked on the DME file. This does that.

Latentish reviewed (changes requested):
This is a fine ruin, but I'm reticent about some of the more dangerous seeds. I'd like to see them included as they can be fun, but kudzu in particular can get out of control easily. So please put some decals or props to indicate its danger. Telegraph the danger.

iwishforducks committed:
Requested changes + nicer things

- Revamps the "secure" area of the lab, telegraphing the danger of kudzu more
- Removes the death nettle seed
- Replaces airlocks to make them telegraph their rooms better
- Replaced the blue tomatoes with blood tomatoes
- Added frames to the in-construction hydroponics room
- Middle room's tray removed, berries added

iwishforducks commented:
Revamped the "vault" room a bit more since I kind of hastily whipped that one up, along with making the requested change.

![image](https://user-images.githubusercontent.com/65363339/177621060-a926a73d-ca85-4049-ab9d-2c68ae9a4008.png)

Made things look nicer in general too, updated the main image in the PR to reflect it, but the takeaway is that airlocks telegraph their rooms better and rooms make a little more sense.

Latentish commented:
This ruin feels like a lootbox. You have a decent spread of enemies but you fill this ruin to the brim with good shit. You'll need to cut back. I'd recommend removing the weapons and security stuff, as this seems to revolve around the good botany loot more.

iwishforducks committed:
Rebalancing

- Removes DNA manipulator
- Removes hydro-chemical synthesizer

iwishforducks commented:
I've opted to just remove the plant DNA manipulator and botanical-chemical synthesizer. The security related loot is pretty negligible and only exist to tell a story - ultimately there is nothing more to cut down on in that regard.

Latentish reviewed (approved):
Pretty much.
TM?

iwishforducks commented:
I think it's fine for a full merge. I got lucky with an ice planet once and it seemed to spawn in just fine. If that's what you're asking, that is, with the whole testmerge question.
